routing-configurer/src/main/java/no/vegvesen/ixn/federation/RoutingConfigurer.java

```diff
@@ -93,9 +95,22 @@ private void setupRouting(List<? extends Subscriber> readyToSetupRouting, String
 		}
 	}
 
-	private void addSubscribersToGroup(String groupName, Subscriber subscriber) {
+	@Scheduled(fixedRateString = "${routing-configurer.groups-interval.add}")
+	private void setupUserInFederationGroup() {
```

Comment:
Maybe change name to 
```suggestion
	private void setupNeighbourInFederationGroup() {
```
for the added clarity that this method only deals with Neighbours.